src/domUpdates.js

```diff
@@ -1,9 +1,32 @@
 let domUpdates = {
+  showSignIn() {
```

Comment:
These functions are looking really clean, I like how you put all of the hiddens into the same function!